src/pages/auth/Login.tsx

```diff
@@ -1,29 +1,43 @@
-import { Box, Copy } from "@/shared/components"
+import React from "react"
+import { Box, GridContainer, Grid, GridItem, Logo, Title } from "@/shared/components"
+import { CommonLayout } from "@/shared/layouts/common"
 
 import { LoginFormContainer } from "./LoginFormContainer"
 
 export const Login = () => {
   return (
-    <Box
-      css={{
-        display: "flex",
-        justifyContent: "center",
-        alignItems: "center",
-        minHeight: "100vh",
-        backgroundColor: "#2363eb",
-      }}
-    >
-      <Box
-        css={{
-          display: "flex",
-          justifyContent: "center",
-          alignItems: "center",
-          flexDirection: "column",
-        }}
-      >
-        <Copy>Login to have access!</Copy>
-        <LoginFormContainer />
-      </Box>
-    </Box>
+    <CommonLayout>
+      <GridContainer fullBleed>
+        <Grid columns={{ "@initial": "1fr", "@sm": "1fr 1fr" }} css={{ "@sm": { height: '100vh' }, height: '100%'}}>
+          <GridItem css={{
+            backgroundColor: "$neutrals-1",
+            padding: '$32 $24',
+            display: "flex",
+            flexFlow: 'column nowrap',
+            justifyContent: 'center'
+          }} >
+            <Box css={{flex: '0 0 auto',}}>
+              <Logo />
+            </Box>
+            <Box
```

Comment:
we can use `<Flex>` here with props like "direction", "justify" etc.

Reply:
Here I need flex property that set flex-shrink, flex-growth, flex-basis, we don't have them in <Flex> component. Should I extand the component?